config/jobs/kubernetes-sigs/kube-storage-version-migrator/ci.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
+  kubernetes-sigs/kube-storage-version-migrator:
+  - name: pull-kube-storage-version-migrator-disruptive
+    decorate: true
+    always_run: true
+    # TODO: remove this when the test is stable
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --check-leaked-resources
+        - --extract=ci/latest
+        - --gcp-master-image=gci
+        - --gcp-node-image=gci
+        - --gcp-nodes=1
+        - --gcp-zone=us-central1-f
+        - --provider=gce
+        - --test=false
+        - --test-cmd='../test/e2e/test-fully-automated.sh --focus="\[Disruptive\]"'
```

Comment:
Can you help double check if this line is correct?

Reply:
This command will be invoked from the checkout of the first repo specified.